.travis-ipython4-requirements.txt

```diff
@@ -0,0 +1,22 @@
+Jinja2>=2.7.3
```

Comment:
I added here traitlets, notebook and nbformat. These files are only needed for IPython 4.0 and break the IPython 3.2 tests.